Tests/SwiftKueryORMTests/TestUpdate.swift

```diff
@@ -0,0 +1,43 @@
+import XCTest
+
+@testable import SwiftKueryORM
+import Foundation
+import KituraContracts
+
+class TestUpdate: XCTestCase {
+    static var allTests: [(String, (TestUpdate) -> () throws -> Void)] {
+        return [
+            ("testUpdate", testUpdate),
+        ]
+    }
+
+    struct Person: Model {
+        static var tableName = "People"
+        var name: String
+        var age: Int
+    }
+
+    func testUpdate() {
+        let connection: TestConnection = createConnection()
+        Database.default = Database(single: connection)
+        performTest(asyncTasks: { expectation in
+            let person = Person(name: "Joe", age: 38)
+            person.update(id: 1) { p, error in
+                XCTAssertNil(error, "Update Failed: \(String(describing: error))")
+                XCTAssertNotNil(connection.query, "Update Failed: Query is nil")
+                if let query = connection.query {
+                  let expectedQuery1 = "UPDATE People SET name = 'Joe', age = 38 WHERE People.id = '1'"
+                  let expectedQuery2 = "UPDATE People SET age = 38, name = 'Joe' WHERE People.id = '1'"
+                  let resultQuery = connection.descriptionOf(query: query)
+                  XCTAssert(resultQuery == expectedQuery1 || resultQuery == expectedQuery2)
```

Comment:
I understand why it is necessary to have the permutations here (unpredictable ordering due to backing by a hashed data structure), but I personally think this would be nicer as follows:
```swift
let expectedPrefix = "UPDATE People SET"
let expectedSuffix = "WHERE People.id = '1'"
let expectedUpdates = ["name = 'Joe'", "age = 38"]
XCTAssertTrue(resultQuery.hasPrefix(expectedPrefix))
XCTAssertTrue(resultQuery.hasSuffix(expectedSuffix))
for update in expectedUpdates {
    XCTAssertTrue(resultQuery.contains(update))
}
```
It's more code, but saves you having to enumerate all the permutations once you have > 2 fields.

Reply:
I agree that is easier to maintain, will change it